### DISCUSSION TITLE
Icons are still hidden after changins settings

### DISCUSSION THREAD

**Discussion by Disp350** [valinet/ExplorerPatcher#1961] [General]
Running Windows 11 22H2 I22621.2134.  have set several icons in the system tray to Show Icons and Notifications, but the still remain hidden.  The only way to display them is to select Allways Show Icons...

Am I missing a step?

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this one.
